### ISSUE TITLE
[Rendering]  Incorrect value for submit Input.ChoiceSet option

### ISSUE THREAD

**Issue by Wajeed-msft** [MicrosoftDocs/msteams-docs#10224]
Copied bug from: https://github.com/microsoft/AdaptiveCards/issues/8626 

**Details from original post:**
### Target Platforms

NodeJS

### SDK Version

1.4

### Application Name

Teams

### Problem Description

1. set a default value(eg: 1,2,3) in Input.ChoiceSet
2. toogle choise 2 ant toogle again

except result:  1,2,3
result: when i get taskModuleRequest.data.requiredMultiselectId.   it is 1,2

it confused me. 

### Screenshots

_No resp

https://github.com/microsoft/AdaptiveCards/assets/38236070/b0a66bf9-0330-4c7b-8c74-63566ed69a6e

onse_

### Card JSON

```JSON
body: [
      {
        type: 'Input.ChoiceSet',
        id: 'requiredMultiselectId',
        style: 'expanded',
        isMultiSelect: true,
        label: 'Manage notifications',
        isRequired: true,
        errorMessage: 'Select at least one items',
        value: defaultValue,
        choices: [
          {
            title: '11111',
            value: 'watcher',
          },
          {
            title: '222222',
            value: 'creator',
          },
          {
            title: '333333',
            value: 'assignee',
          },
          {
            title: '44444',
            value: 'comment_mentioned',
          },
        ],
      },
      {
        type: "Input.Text",
        placeholder: "card_type",
        id: 'card_type',
        isVisible: false,
        value: TaskSubmitType.NotifyChange
      },
      {
        type: 'Container',
        items: [
          {
            type: 'ActionSet',
            actions: [
              {
                type: 'Action.Submit',
                title: 'Update',
              },
            ],
            horizontalAlignment: 'right',
          },
        ],
      },
    ],
```

### Sample Code Language

en

### Sample Code

_No response_---
#### Document Details

⚠ *Do not edit this section.It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3fb4394d-25b1-8752-3996-fe3a2934732d
* Version Independent ID: 6d391381-89d0-117c-27cc-00043bd26a36
* Content: [Cards reference - Teams](https://docs.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/cards-reference)
* Content Source: [msteams-platform/task-modules-and-cards/cards/cards-reference.md](https://github.com/MicrosoftDocs/msteams-docs/blob/master/msteams-platform/task-modules-and-cards/cards/cards-reference.md)
* Product: **msteams**
* GitHub Login: @surbhigupta12
* Microsoft Alias: **surbhigupta12**

**Comment by ChetanSharma-msft** [comment]
Hello @86driver - Thanks for raising your query.
We will look into it and let you know the updates.

**Comment by Dinesh-MSFT** [comment]
Hi @86driver,

We have tried this JSON.
[Card JSON.txt](https://github.com/MicrosoftDocs/msteams-docs/files/14051268/Card.JSON.txt)

When we open the card and click on submit. It is sending a default selected value as response.
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/154973237/eb6f14f6-0e6e-46ae-86a1-5f129467c9e4)

But, later after opening the same card without refreshing, if we selected any other option along with default value and submit it. The response is submitting as only for selected value instead of both selected value and default value.
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/154973237/ecf2e803-53a8-4ac5-8b05-9dc3cb7b3755)

Here, we are checking with engineering team whether it is a by design feature or a bug. We will look into it and let you know the updates.
